scripts/prometheus/config/grafana/dashboards/oe.json

```diff
@@ -0,0 +1,1576 @@
+{
```

Comment:
Are these configs something we really want to ship inside the repo? cc @denisgranha